integration_tests/tests/consistency/consistency_email_sends.sql

```diff
@@ -3,7 +3,7 @@
     enabled=var('fivetran_validation_tests_enabled', false)
 ) }}
 
-{% set columns_to_exclude = ['activity_id'] %}
+{% set columns_to_exclude = var('consistency_test_exclude_columns', []) %}
```

Comment:
I think `activity_id` would always be excluded?
```suggestion
{% set columns_to_exclude = ['activity_id'] + var('consistency_test_exclude_columns', []) %}
```

Reply:
Hmm the order of operations is still off I think. Line 12 would overwrite the work of line 9? I tink you just need to replace line 6 with line 12, and remove line 12.

Reply:
Doh yeah, updating!